data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/main/ConsumerVerticleFactoryImpl.java

```diff
@@ -38,7 +38,7 @@
 import dev.knative.eventing.kafka.broker.dispatcher.impl.consumer.OffsetManager;
 import dev.knative.eventing.kafka.broker.dispatcher.impl.consumer.OrderedConsumerVerticle;
 import dev.knative.eventing.kafka.broker.dispatcher.impl.consumer.UnorderedConsumerVerticle;
-import dev.knative.eventing.kafka.broker.dispatcher.impl.filter.AttributesFilter;
+import dev.knative.eventing.kafka.broker.dispatcher.impl.filter.*;
```

Comment:
should we do explicit imports? 

I am not a fan of `*` wildcard imports

Reply:
+1

Reply:
there is a setting in intellij idea to increase the number of imports to make it a star, I set it to 9999 to basically disable that feature.

Reply:
> I set it to 9999 to basically disable

classic :smile: